pkg/certificate/manager.go

```diff
@@ -12,7 +13,7 @@ import (
 )
 
 // NewManager creates a new CertManager with the passed CA and CA Private Key
-func NewManager(mrcClient MRCClient, serviceCertValidityDuration time.Duration, msgBroker *messaging.Broker) (*Manager, error) {
+func NewManager(mrcClient MRCClient, getServiceCertValidityPeriod func() time.Duration, msgBroker *messaging.Broker) (*Manager, error) {
```

Comment:
Could we move the validity period to the MRC itself? Since there's already a mechanism for updating that